libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -69,6 +74,32 @@ class AC_PrecLand
     // process a LANDING_TARGET mavlink message
     void handle_msg(const mavlink_landing_target_t &packet, uint32_t timestamp_ms);
 
+    // State of the Target Location
+    enum class State: uint8_t {
+        TARGET_NEVER_SEEN = 0,
+        TARGET_OUT_OF_RANGE,
+        TARGET_RECENTLY_LOST,
+        TARGET_FOUND
+    };
+
+    // return the last time PrecLand library had a output of the landing target position
+    uint32_t get_last_precland_output_ms() const { return _last_precland_output_ms; }
+
+    // return the current state of the location of the landing target
+    State get_plnd_target_state() const { return _current_target_state; }
+
+    // return the last known landing location in Earth Frame NED cm.
+    void get_last_detected_landing_location(Vector3f &loc) const { loc = _last_target_loc_rel_origin_NED; }
```

Comment:
let's rename this to "get_last_detected_landing_position()" because it returns a Vector3f instead of a Location. the argument should also be renamed from "loc" to "pos".

Reply:
NED cm is very odd.. is that correct?  normally it is NED m or NEU cm.